src/libraries/Microsoft.PowerFx.Interpreter/Marshal/TypeCoercionProvider.cs

```diff
@@ -0,0 +1,109 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Globalization;
+using System.Threading;
+using Microsoft.PowerFx.Core.IR;
+using Microsoft.PowerFx.Functions;
+using Microsoft.PowerFx.Types;
+using static Microsoft.PowerFx.Functions.Library;
+
+namespace Microsoft.PowerFx
+{
+    /// <summary>
+    /// Convert value to target format.
+    /// </summary>
+    public static class TypeCoercionProvider
+    {
+        public static FormattingInfo CreateFormattingInfo()
```

Comment:
>public [](http://example.com/codeflow?start=8&length=6)

We can make this internal.
Then that lets us leave FormattingInfo class as internal.

Reply:
udpated.